### PR TITLE
feat(hardware-testing): Create a way of storing patches in hardware-testing

### PR DESCRIPTION
**Pull request by ryanthecoder**
# Overview
Hardware-testing gravimetric tests need certain things patched in the api and shared-data directories
before we kept a seperate DO-NOT-MERGE branch around and had to rebase onto to make changes to the script than create another branch off of that to merge those changes to edge and it was pretty clunky. This allows testing to create, and apply patches easily in the main trunk without needing to maintain an extra branch

# Test Plan

# Changelog

# Review requests

# Risk assessment

### CODE REVIEW

**Review by andySigler — APPROVED** [review]

File: hardware-testing/Makefile [hunk 1]
```diff
@@ -220,3 +220,26 @@ sync-fw-ot3:
 
 .PHONY: sync-ot3
 sync-ot3: sync-sw-ot3 sync-fw-ot3
+
+.PHONY: push-ot3-gravametric
```
Comment (sfoster1) [inline hardware-testing/Makefile]:
```suggestion
.PHONY: push-ot3-gravimetric
```

Comment (sfoster1) [reply hardware-testing/Makefile]: and so on

**Review by sfoster1 — CHANGES_REQUESTED** [review]
ah, this is way better than maintaining a separate branch. i have two requests, however:
- let's add a CI flow triggered on changes to the things the patches change that tries to apply the patches and yells at you if it fails, so we don't end up having problems where suddenly other changes broke the patches and thus the gravimetric test process
- spelling

**Review by sfoster1 — APPROVED** [review]
Okay, looks good to me!

**Pull request merged**